java-frontend/src/main/java/org/sonar/java/se/symbolicvalues/SymbolicValue.java

```diff
@@ -155,6 +156,19 @@ public SymbolicValue wrappedValue() {
     return this;
   }
 
+  public static class TypedSymbolicValue extends SymbolicValue {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2160)